pycqed/measurement/calibration_points.py

```diff
@@ -19,10 +19,12 @@ def __init__(self, qb_names, states, **kwargs):
         self.states = states
```

Comment:
Thanks for fixing this issue :) I just have a few questions/comments on the implementation (see comments below)